mozci/queries/push_tasks_from_task.query

```diff
@@ -0,0 +1,13 @@
+from: task
+select:
+    - {name: id, value: task.id}
+    - {name: kind, value: task.kind}
+    - {name: tags, value: task.tags}
```

Comment:
What were you planning to do with the kind and tags info? I'm not too familiar with the fields, just saw an example now with https://firefox-ci-tc.services.mozilla.com/api/queue/v1/task/crJUWRrkQN-V7N1IJUV6Kg.

Nit: I see `kind` is part of `tags`, maybe we can just retrieve `tags`?